src/core/zowe/core_for_zowe_sdk/config_file.py

```diff
@@ -128,14 +135,46 @@ def init_from_file(self) -> None:
             profile_jsonc = commentjson.load(fileobj)
 
         self.profiles = profile_jsonc.get("profiles", {})
+        self.schema_property = profile_jsonc.get("$schema", None)
         self.defaults = profile_jsonc.get("defaults", {})
         self.schema_property = profile_jsonc.get("$schema", None)
 
+        if self.schema_property:
+            self.validate_schema(config_type, validate_schema, verify=verify)
         # loading secure props is done in load_profile_properties
         # since we want to try loading secure properties only when
         # we know that the profile has saved properties
         # self.load_secure_props()
 
+    def validate_schema(
+        self,
+        config_type: str,
+        validate_schema: Optional[bool] = False,
```

Comment:
Is the `validate_schema` option required on this function? If the`validate_schema`method is called, I would expect it to always validate the schema.